AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.java

```diff
@@ -1032,9 +1034,15 @@ public void onBackPressed() {
             if (mFloatingActionMenu.isFABOpen()) {
                 mFloatingActionMenu.closeFloatingActionMenu();
             } else {
-                automaticSync();
-                finishWithAnimation();
+                if (mBackButtonPressedToExit) {
+                    automaticSync();
+                    finishWithAnimation();
+                }
+                else {
+                    UIUtils.showThemedToast(this, getString(R.string.back_pressed_once), true);
+                }
             }
+            mBackButtonPressedToExit = true;
```

Comment:
Only set this inside the `else` block
```suggestion
            mBackButtonPressedToExit = true;
```